.github/workflows/ci.yml

```diff
@@ -293,8 +293,9 @@ jobs:
         include:
           - test: saw-remote-api/scripts/run_rpc_tests.sh
             os: ubuntu-18.04
-          - test: saw-remote-api/scripts/run_rpc_tests.sh
-            os: macos-12
+          # TODO: These tests are disabled pending resolution of issue #1699
+          # - test: saw-remote-api/scripts/run_rpc_tests.sh
+          #   os: macos-12
```

Comment:
Rather than disabling this step wholesale, I think it would be more reasonable to change [these lines](https://github.com/GaloisInc/saw-script/blob/f71a795ea0daae8226b1c9c5c5f19d062ce1008d/saw-remote-api/scripts/run_rpc_tests.sh#L36-L37) to something like:

```bash
if [[ "$RUNNER_OS" != "macOS" ]]; then
  run_test poetry run python -m unittest discover tests/saw
  run_test poetry run python -m unittest discover tests/saw_low_level
fi
```

That way, we still ensure that we can at least build the dependencies and typecheck them. Based on the results of the investigation thus far in #1701, that should still be feasible even with #1699 occurring.

Reply:
I agree that we should probably still build and typecheck dependencies, however I'm somewhat hesitant to add a CI-specific path to `run_rpc_tests.sh` - at a glance, it looks like that script is maybe intended to be used as a general-purpose entrypoint to the `saw-remote-api` test suite. I'll poke around and see if I can find something that feels nice.

Reply:
Very true. I suggested the change above since you could still run `run_rpc_tests.sh` locally on macOS and have it run, provided that you didn't define the `RUNNER_OS` environment variable. But I see your point about the change being invasive. As an alternative, we could split up `run_rpc_tests.sh` into smaller scripts and skip the one that runs the unit tests only on macOS.